addons/event/tests/test_event_flow.py

```diff
@@ -14,30 +14,32 @@
 class TestEventUI(TestEventCommon):
 
     def test_event_registration_partner_sync(self):
-        """ Ensure onchange on partner_id is kept for interface, not for computed
```

Comment:
Actually ``test_registration_partner_sync`` already checks most of the compute, if not everything. If we remove the onchange, I feel like this whole test could be removed as it is linked only with the ochange. You should probably tell it is a revert of odoo/odoo@c6003af73409e2cd124f641fb7714fefa9dc8b01 since we feel this is not necessary anymore.